TestGeneration.Extensions.NUnit/source.extension.vsixmanifest

```diff
@@ -7,8 +7,8 @@
 Creates Unit tests and Intellitests with both NUnit 2.6.4 and NUnit 3 frameworks.
 </Description>
     <MoreInfo>https://github.com/nunit/nunit-vs-testgenerator</MoreInfo>
-    <GettingStartedGuide>https://github.com/nunit/docs/wiki/Visual-Studio-Test-Generator</GettingStartedGuide>
     <License>license.txt</License>
+    <GettingStartedGuide>https://github.com/nunit/docs/wiki/Visual-Studio-Test-Generator</GettingStartedGuide>
```

Comment:
I couldn't build without re-ording the vsixmanifest elements like this. I added the <GettingStartedGuide> tag in another PR, but I didn't think there was any required ordering in vsixmanifests. Do you know if there is?